sdk/src/types/block/unlock/signature.rs

```diff
@@ -27,6 +30,12 @@ impl SignatureUnlock {
     }
 }
 
+impl WorkScore for SignatureUnlock {
+    fn work_score(&self, params: WorkScoreParameters) -> u32 {
+        params.signature_ed25519()
```

Comment:
This needs to forward to the signature work score

Reply:
Fixed